docs/website/docusaurus.config.js

```diff
@@ -191,6 +191,10 @@ const config = {
                 label: 'Mithril Networks Status',
                 href: 'https://mithril.cronitorstatus.com/'
```

Comment:
This looks nice too! Is this a free service?

Reply:
Yes, a freemium service: https://cronitor.io/